lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOFavoriteDAOImplAutowiredTest.java

```diff
@@ -263,7 +265,7 @@ public void testGetFavoritesEntityHeader() throws Exception {
 		Favorite fav2created = favoriteDao.add(fav2);
 		favoritesToDelete.add(fav2created);
 
-		PaginatedResults<EntityHeader> favs = favoriteDao.getFavoritesEntityHeader(creatorUserGroupId.toString(), Integer.MAX_VALUE, 0);
+		PaginatedResults<EntityHeader> favs = favoriteDao.getFavoritesEntityHeader(creatorUserGroupId.toString(), Integer.MAX_VALUE, 0, SortBy.FAVORITED_ON, SortDirection.ASC);

```

Comment:
How do you know that fav2created will have a later timestamp than fav1created?  They could be created in the same millisecond, have the same time stamp, and then the outcome of this test becomes random.  Either mock the clock in the DAO or put a > 1 ms delay between creation of the two favorites.